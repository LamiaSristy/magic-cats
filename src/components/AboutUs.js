import React from 'react';
import Navbar from "./Navbar";
import Footer from "./Footer";
import '../stylesheets/AboutUs.scss'

const AboutUs = () => (
  <div class="author-container">
    <Navbar />
    <div className="author-detail-container">  
      <p className="author-heading">We are Cat Lovers ðŸ’•!!</p>
      <p>The cat (Felis catus) is a domestic species of small carnivorous mammal.[1][2] It is the only domesticated species in the family Felidae and is often referred to as the domestic cat to distinguish it from the wild members of the family.[4] A cat can either be a house cat, a farm cat or a feral cat; the latter ranges freely and avoids human contact.[5] Domestic cats are valued by humans for companionship and their ability to hunt rodents. About 60 cat breeds are recognized by various cat registries.[6]</p>
      <p>The cat is similar in anatomy to the other felid species: it has a strong flexible body, quick reflexes, sharp teeth and retractable claws adapted to killing small prey. Its night vision and sense of smell are well developed. Cat communication includes vocalizations like meowing, purring, trilling, hissing, growling and grunting as well as cat-specific body language. A predator that is most active at dawn and dusk, the cat is a solitary hunter but a social species. It can hear sounds too faint or too high in frequency for human ears, such as those made by mice and other small mammals.[7] It secretes and perceives pheromones.[8]</p>
      <p>Female domestic cats can have kittens from spring to late autumn, with litter sizes often ranging from two to five kittens.[9] Domestic cats are bred and shown at events as registered pedigreed cats, a hobby known as cat fancy. Failure to control breeding of pet cats by spaying and neutering, as well as abandonment of pets, resulted in large numbers of feral cats worldwide, contributing to the extinction of entire bird, mammal, and reptile species, and evoking population control.[10]</p>
    </div>
    <Footer />
  </div>
  
);

export default AboutUs;
