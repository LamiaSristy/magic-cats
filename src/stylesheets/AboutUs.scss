@import '~bootstrap/scss/bootstrap.scss';

.author-detail-container {
  text-align: center;
  font-family: Source Sans Pro, sans-serif;
  color: rgb(216, 46, 75);
  background-color: rgb(207, 203, 203);
  padding: 60px 15px;
  margin-top: 30px;
  margin-left: 30px;
  margin-right: 30px;
  border: 1px dotted rgb(216, 46, 75);
  box-shadow: 0 0.2rem 0.5rem #e26ca3;
  border-radius: 5px;
}

.author-heading {
  font-size: x-large;
}
