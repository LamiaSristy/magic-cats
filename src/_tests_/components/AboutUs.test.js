import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import AboutUs from '../../components/AboutUs';

const setup = () => shallow(<AboutUs />);

describe('AboutUs component', () => {
  let component;

  beforeEach(() => {
    component = setup();
  });

  it('Should render', () => {
    const element = component.find('.author-heading');
    expect(element.length).toEqual(1);
  });

  test('renders the A heading', () => {
    const heading = component.find('.author-heading');
    expect(heading).toMatchSnapshot();
  });
});
