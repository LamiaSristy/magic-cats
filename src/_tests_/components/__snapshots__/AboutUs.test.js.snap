// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component renders the A heading 1`] = `ShallowWrapper {}`;
